package.json

```diff
@@ -35,34 +35,18 @@
 		"@babel/preset-env": "7.4.2",
 		"@babel/preset-react": "7.0.0",
 		"@babel/runtime": "7.4.2",
-		"@wordpress/a11y": "2.0.2",
-		"@wordpress/api-fetch": "2.2.8",
```

Comment:
And here I thought I caught another usage...

https://github.com/Automattic/wp-calypso/blob/cc97d4fc725cee6bd7f972c9a142edc8316168f9/packages/calypso-build/webpack/wordpress-externals.js#L8

(feel free to remove or replace that comment to avoid confusion)